### ISSUE TITLE
Enable GPT-Neo-125M for downstream training Effectively

### ISSUE THREAD

**Issue by bhadreshpsavani** [EleutherAI/gpt-neo#228]
## Issue while finetuning GPT-Neo-125M on Downstream task

We have a GPT-Neo model with comparatively fewer parameters that can be used for the downstream tasks. It can understand longer sequences.

Currently, it can't be trained in Mini batches because of no padding token in the tokenizer used during pretraining because of that I was facing an [issue](https://github.com/huggingface/transformers/issues/12022)

### Solution:
https://github.com/huggingface/transformers/issues/12022#issuecomment-854711470
